contract-sdk/src/testing.rs

```diff
@@ -51,13 +57,23 @@ impl PublicStore for MockStore {}
 impl ConfidentialStore for MockStore {}
 
 /// Mock environment.
-#[derive(Clone, Default)]
-pub struct MockEnv {}
+#[derive(Clone)]
+pub struct MockEnv {
+    rng: Arc<Mutex<XorShiftRng>>,
+}
 
 impl MockEnv {
     /// Create a new mock environment.
     pub fn new() -> Self {
-        Self {}
+        Default::default()
+    }
+}
+
+impl Default for MockEnv {
+    fn default() -> Self {
+        Self {
+            rng: Arc::new(Mutex::new(XorShiftRng::seed_from_u64(0))),
```

Comment:
Should this be deterministic?

Reply:
it helps if you need to hardcode values into tests or something